sbol/test/test_identified.py

```diff
@@ -4,6 +4,9 @@
 import rdflib
 
 import sbol
+import sbol.config
+import sbol.property
+import sbol.constants
```

Comment:
These lines are unnecessary. Just import `sbol` and use everything from there. No need to separately import other files from the sbol package.

Reply:
Fixed